### PR TITLE
Publish 550.54.15 CUDA drivers

### PR DESCRIPTION
**Pull request by ganeshkumarashok**
For modpost error, new features and other fixes.

Release notes for the 550.54.15 driver: https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-550-54-15/index.html

Modpost issue tracking:
https://forums.developer.nvidia.com/t/linux-6-7-3-545-29-06-550-40-07-error-modpost-gpl-incompatible-module-nvidia-ko-uses-gpl-only-symbol-rcu-read-lock/280908/4

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        driver_version: ["470.82.01", "510.47.03", "515.65.01", "525.85.12", "535.54.03"]
+        driver_version: ["470.82.01", "510.47.03", "515.65.01", "525.85.12", "535.54.03", "550.54.15"]
```
Comment (ganeshkumarashok) [inline .github/workflows/ci.yaml:13]: After testing, we can remove the older driver version

Comment (cameronmeissner) [reply .github/workflows/ci.yaml:13]: can we add an explicit TODO?

Comment (lilypan26) [reply .github/workflows/ci.yaml:13]: is the "older driver version" referring to the 470 driver? We would need to update the driver selection logic too in that case right? Might want to note that in the TODO as well.

File: justfile [hunk 2]
```diff
@@ -10,11 +10,12 @@ cuda_510_driver := "510.47.03"
 cuda_470_driver := "470.82.01"
 cuda_515_driver := "515.65.01"
 cuda_535_driver := "535.154.05"
```
Comment (djsly) [inline justfile:12]: can you bump 535 to the latest swell, in case we need to quickly revert AB back to 535.

Comment (djsly) [reply justfile:12]: https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-535-161-08/index.html

Comment (ganeshkumarashok) [reply justfile]: Updated

Comment (ganeshkumarashok) [reply justfile]: Looks like corresponding fabric manager version was not published for 535 -first time I'm seeing this: https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-x86_64/

Comment (ganeshkumarashok) [reply justfile]: Splitting 535 update to a separate PR, in case we want to revert it later: https://github.com/Azure/aks-gpu/pull/33

**Pull request merged**